Hunting Queries/SecurityEvent/UserAccountCreatedDeleted.txt

```diff
@@ -0,0 +1,33 @@
+// Name: User Account Created and Deleted within 10mins
+// Description: User account created and then deleted within 10 minutes across last 30 days
+//
+// Id: 6135a90e-ba30-4f36-9b6a-3a350050704b
+//
+// QueryPeriod: -30d
```

Comment:
not needed for hunting... if we leave this in then no minus sign.

Reply:
removed